Integration Tests Fail Due to Hardcoded contractimport! Path and Invalid Import

### Description
Running integration tests with:

```bash
cargo test --test integration
```

fails while compiling sweep_controller with the following errors:

Hardcoded absolute path in contractimport!

error: The system cannot find the path specified. (os error 3)
--> contracts\sweep_controller\src\lib.rs:232:5
= note: this error originates in the macro `soroban_sdk::contractimport`


The macro references an absolute filesystem path:

/home/levai/bridgelet-core/target/wasm32-unknown-unknown/...


This breaks compilation on other machines (e.g. Windows) and CI environments.

Unresolved import

error[E0432]: unresolved import `crate::ephemeral_account::Client`
no `Client` in `ephemeral_account`

### Expected Behavior

cargo test --test integration should compile and run successfully across environments.
No absolute local filesystem paths should be required.
Cross-contract client imports should resolve correctly.

### Actual Behavior

Compilation fails before tests run due to:

- Non-portable absolute path in soroban_sdk::contractimport!
- Incorrect or mismatched client import for ephemeral_account

### Environment

- Workspace build (bridgelet-core)
- Rust + Soroban SDK

### Likely Root Causes

contractimport! is pointing to a developer-specific build path instead of a workspace-relative or reproducible path.
The ephemeral_account contract client may not be exported or imported correctly within the workspace structure.

### Suggested Fix Direction

Make contractimport! use a portable, workspace-relative path or alternative mechanism.
Ensure ephemeral_account exposes the correct client type and is imported consistently.

### ⚠️ Contributor Note

This issue involves macro usage and cross-contract integration details. If using AI tools to assist, please
Verify generated suggestions against Soroban SDK documentation.
Avoid blindly pasting path or import fixes without confirming workspace layout.
Ensure the final solution is portable and CI-safe.
Subtle macro and build-path mistakes can compile locally but fail elsewhere.

--- a/contracts/sweep_controller/Cargo.toml
+++ b/contracts/sweep_controller/Cargo.toml
@@ -9,12 +9,12 @@ crate-type = ["cdylib", "rlib"]
 [dependencies]
 soroban-sdk = "22.0.0"
 bridgelet-shared = { path = "../shared", version = "0.1.0" }
+ephemeral_account = { path = "../ephemeral_account", version = "0.1.0" }
 
 soroban-token-sdk = "22.0.0"
 
 [dev-dependencies]
 soroban-sdk = { version = "22.0.0", features = ["testutils"] }
-ephemeral_account = { path = "../ephemeral_account" }
 
 
 [profile.release]

--- a/contracts/sweep_controller/src/lib.rs
+++ b/contracts/sweep_controller/src/lib.rs
@@ -5,7 +5,7 @@ mod errors;
 mod storage;
 // mod transfers;
 
-use crate::ephemeral_account::Client as EphemeralAccountClient;
+use ephemeral_account::EphemeralAccountContractClient as EphemeralAccountClient;
 use soroban_sdk::{contract, contractimpl, contracttype, Address, BytesN, Env};
 
 use authorization::AuthContext;
@@ -224,12 +224,4 @@ fn emit_destination_updated(env: &Env, old_destination: Option<Address>, new_des
     };
     env.events()
         .publish((soroban_sdk::symbol_short!("dest_upd"),), event);
-}
-
-// Re-export ephemeral_account types for cross-contract calls
-mod ephemeral_account {
-    // Import from the actual ephemeral_account contract
-    soroban_sdk::contractimport!(
-        file = "/home/levai/bridgelet-core/target/wasm32-unknown-unknown/release/ephemeral_account.wasm"
-    );
 }

